packages/core/src/exporter.ts

```
@@ -245,138 +172,147 @@ export const fillValue = async (
-        _id: item.parentCompanyId
+        _id: item.parentCompanyId,
       });
 
-      value = parent ? parent.primaryName : "";
+      value = parent ? parent.primaryName : '';
 
       break;
 
-    case "tag":
+    case 'tag':
       const tags = await models.Tags.find({ $in: item.tagIds || [] });
```

Biome

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)